### ISSUE TITLE
nginx and wildcard projects

### ISSUE THREAD

**Issue by ko-christ** [sayakb/sticky-notes#180]
Hello, 

this apache config (http://paste.kde.org/pxfw8gve1) works fine for subdomain projects (and SSL redirection). 

This nginx config (http://paste.kde.org/pu3jhd2dn) works fine for SSL redirection, but subdomains (for sticky projects) does not work properly. 
My first subdomain e.g. aaa.paste.my.tld works fine, creates correctly the autogenerated URLS 
(https://aaa.paste.my.tld/all?/all https://aaa.paste.my.tld/trending docs etc)

But all subsequent requests for new subdomains (e.g. bbb.paste.my.tld) have links to the first one (https://aaa.paste.my.tld/trending) until I manually clear cache. (php artisan cache:clear).
In short the above configuration can handle only one subdomain. All subsequent subdomains link to the first one.

Is there a working nginx configuration with subdomains for projects?

Thanks.

**Comment by sayakb** [comment]
I finally have a new box, but I don't have any way to set up nginx on it. I wont be able to look into this.
